docs/seps/sep-0005.rst

```diff
@@ -36,10 +36,13 @@ The current proposed structure of a timeseries is that of Python `dictionary`, r
 
   {
         "data": np.array([[1,2,3,4], [5,6,7,8]]),         # Array of data shaped (n,) (or (m, n) for `m` channels) and `n` time-samples.
-        "time": np.array([1,3,4,7]),                      # Time-vector in seconds. Optional for equally spaced data.
-        "fs": 125,                                        # Sampling frequency in Hz. Optional if `time` specified.
-        "name": "timeseries_id",                          # Timeseries name 
-        "channel_name",                                   # Optional: Name of the measured physical channel (or list of `m` channel names if `data` is of shape (m, n)).
+        "fs": float(125),                                 # Sampling frequency in Hz. Optional if `time` specified.
+        "name": "structure xyz",                          # Timeseries name 
+        "time": np.array([1,3,4,7]),                      # Optional for equally spaced data: time-vector in seconds (shaped (n,)).
+        "channel_name": ["1_x", "1_y"],                   # Optional: Name of the measured physical channel (or list of `m` channel names if `data` is of shape (m, n)).
+        "channel_index": np.array([0,1]),                 # Optional: Index of channels used (array of `m`). To bi used with channel object (to be defined)
```

Comment:
Is this needed? A list of channel names for each of the data rows is how we plan to keep track of this now, and it's a requirement that this has the same length than our `data`. Wouldn't the `channel_index` then always be just equal to `np.arange(len(data))`?

Reply:
`channel_index` can be removed, thanks.